android/jni/app-android.cpp

```diff
@@ -1011,9 +1012,12 @@ extern "C" bool JNICALL Java_org_ppsspp_ppsspp_NativeActivity_runEGLRenderLoop(J
 		ELOG("Error: Surface is null.");
 		return false;
 	}
-	
+
+retry:
+
 	bool vulkan = g_Config.iGPUBackend == GPU_BACKEND_VULKAN;
 
+	int tries = 0;
```

Comment:
Hm, not really familiar, what happens with goto in this scenario?  I assume `tries` isn't reset to 0, right?

-[Unknown]